### PR TITLE
2064 add new page for fee paying or salaried

### PR DESCRIPTION
**Pull request by defong**
### Context
New courses

### Changes proposed in this pull request
Add "fee or salary" selection for new course flow

### Guidance to review
🚢 

image

### CODE REVIEW

**Comment by defong** [comment]
[Add "new" page for Fee paying or salaried](https://trello.com/c/LdiuA9I4/2064-add-new-page-for-fee-paying-or-salaried)

File: app/views/courses/fee_or_salary/_form_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  data-qa="course__program_type">
+
+  <fieldset class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+      <h1 class="govuk-fieldset__heading">
+        Is it fee paying or salaried?
+      </h1>
+    </legend>
+
+    <%= render "shared/error_messages", field: :program_type if @errors %>
```
Comment (misaka) [inline app/views/courses/fee_or_salary/_form_fields.html.erb]: I had another look at this tonight, with @faissalMT, and after the chat with @dankmitchell, and I'm starting to think that this is in fact the wrong shape. Translating the `program_type` to whether the course is fee paying or salaried should be in the backend, not here, and the API should reflect that. I think we need to have another look at this.

Comment (defong) [reply app/views/courses/fee_or_salary/_form_fields.html.erb]: I am also doubt that exposing the current course model is the right approach. It feels like another in direction should be in place. May be a facade, that is the final replacement for the current course model to be persisted or an adaptor.

Comment (defong) [reply app/views/courses/fee_or_salary/_form_fields.html.erb]: @davidgisbey fyi

**Pull request merged**